packages/sit-onyx/src/components/OnyxCalendar/OnyxCalendar.vue

```diff
@@ -161,25 +193,32 @@ const getDayRangeType = computed(() => {
   >
     <div class="onyx-calendar__header">
       <div class="control-container time-control-container">
-        <OnyxTag
-          v-if="calendarSize !== 'small'"
+        <OnyxSystemButton
           :label="t('calendar.todayButton.label')"
           class="control-container__today-btn"
           :disabled="disabled === true"
           :clickable="t('calendar.todayButton.tooltip')"
           @click="goToToday"
         />
+
+        <OnyxHeadline is="h2" class="control-container__date-display">
+          {{ d(viewMonth, { month: "long", year: "numeric" }) }}
+        </OnyxHeadline>
+        <OnyxTag
+          v-if="showCalendarWeeks && calendarSize === 'big'"
+          color="primary"
+          :label="`${t('calendar.calenderWeek')} ${weeksToRender[0]?.weekNumber} - ${weeksToRender[weeksToRender.length - 1]?.weekNumber} `"
```

Comment:
```suggestion
          :label="`${t('calendar.calenderWeek')} ${weeksToRender[0]?.weekNumber} - ${weeksToRender.at(-1)?.weekNumber} `"
```

Maybe we should also extract this to a dedicated computed?